var/spack/repos/builtin/packages/opencv/package.py

```diff
@@ -891,30 +891,38 @@ def determine_version(cls, lib):
             pattern = None
```

Comment:
Let's remove the ver and pattern variables, they aren't needed

Reply:
done